src/Common/src/System/Net/SafeCloseSocket.Windows.cs

```diff
@@ -117,6 +117,11 @@ private void InnerReleaseHandle()
             }
         }
 
+        private void TryWakeup(InnerSafeCloseSocket innerSocket)
+        {
+            return;
```

Comment:
Nit: this isn't necessary; you can just remove the "return;" and leave the method empty.  You might also add a comment indicating that it's empty on purpose, and explaining why

Reply:
I may need to revisit this. comments from the bugs suggest that Windows may have problems in single core configuration. I plan to test it but I was using this PR to get wide test coverage from CI run.